news/2017-08-17-happystick-summer-tour-concludes.md

```diff
@@ -0,0 +1,50 @@
+
+[Full match information](https://osu.ppy.sh/community/matches/35247144)
+
+<p align="center"><iframe src="https://player.twitch.tv/?video=v167002704&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no" height="378" width="620"></iframe></p>
+
+Fans turned up in droves to watch the [1000 - 3000 bracket](http://challonge.com/a36241q1) Grand Final featuring fan favorite [Lee Jung Hyun](https://osu.ppy.sh/u/4065138) (affectionately nicknamed “Shakeman” for his almost bizarrely shaky playstyle that belied impeccable aim) against [\_verto\_](https://osu.ppy.sh/u/2015300), who quietly made his way through the Winners Bracket while losing only 5 maps to make it to the Grand Final. However, Lee Jung Hyun was not to be trifled with, as shown by victories in the Losers Bracket over multiple strong opponents. In the Grand Final, the players utilized the entire mappool, as both showcased their diverse skillsets by selecting maps from every mod. After trading map wins to start the match, Lee Jung Hyun took the first point on an opponent’s pick by winning on [MY FIRST STORY - ALONE](https://osu.ppy.sh/b/861381) (in spite of [shakes normally reserved for locations of an ongoing earthquake](https://www.twitch.tv/videos/167002704?t=0h32m00s) on the ending jumps) to take a 3-1 lead, then each player won his own pick for the next 4 maps, giving Lee set point at 5-3. Selecting notoriously difficult alternating map [HujuniseikouyuuP - Talent Shredder](https://osu.ppy.sh/b/153857), Hyun capitalized and took the first set 6-3, sending the players to a bracket reset match. The second set was a familiar sight as both players won their early picks until Lee Jung Hyun took a 4-2 lead by taking advantage of an agonizing last-second fail on the [absolute last slider tick](https://www.twitch.tv/videos/167002704?t=01h25m55s) of Hard Rock pick [Nardis - Cosmo Memory](https://osu.ppy.sh/b/1099759) from \_verto\_, then repeated his Talent Shredder success to reach championship point at 5-2. There, \_verto\_ selected [Igorrr - Unpleasant Sonata](https://osu.ppy.sh/b/245284) from the Freemod pool. Both players held strong combos for much of the map until \_verto\_ broke two-thirds of the way through. Holding on for another 60 combo, Lee Jung Hyun took the lead and held on to win by under one million points to win the match 6-2.
```

Comment:
new paragraph at 'in the grand final'

new paragraph at 'the second set'

new paragraph at 'There, \_verto\_